src/Request.php

```diff
@@ -111,9 +111,19 @@ public function hasHeader($name)
         return !!$this->getHeader($name);
     }
 
+    /**
+     * Checks if the request headers contain the `Expect: 100-continue` header.
+     *
+     * This header MAY be included when an HTTP/1.1 client wants to send a bigger
+     * request body.
+     * See [`writeContinue()`] for more details.
+     *
+     * @return bool
+     * @see Response::writeContinue()
+     */
     public function expectsContinue()
     {
-        return isset($this->headers['Expect']) && '100-continue' === $this->headers['Expect'];
+        return '100-continue' === $this->getHeaderLine('Expect');
```

Comment:
This needs a `strtolower` call for `$this->getHeaderLine("expect")`, see https://tools.ietf.org/html/rfc7231#section-5.1.1

> The Expect field-value is case-insensitive.